ProjectSettings/ProjectSettings.asset

```diff
@@ -621,7 +621,7 @@ PlayerSettings:
   webGLThreadsSupport: 0
   webGLWasmStreaming: 0
   scriptingDefineSymbols:
-    1: UNITY_ENABLE_STEAM_CONTROLLER_SUPPORT
+    1: UNITY_ENABLE_STEAM_CONTROLLER_SUPPORT;UNITY_INPUT_SYSTEM_CONTROL_VALUE_CACHING
```

Comment:
Intentional I assume.
Would this mean CI and/or local TestRunner tests would have this enabled?
What about packaged Sample projects. Would they be affected?
Any other implications?